cmd/kubeadm/app/phases/addons/dns/dns.go

```diff
@@ -273,16 +274,17 @@ func createDNSService(dnsService *v1.Service, serviceBytes []byte, client client
 		return fmt.Errorf("unable to decode the DNS service %v", err)
 	}
 
+	ctx := context.TODO()
```

Comment:
to simplify the PR, can we just use `context.TODO()` directly in all calls everywhere? I.e. no local ctx. We can (should!) have follow-up add the proper context logic. But for getting it type-correct, `context.TODO()` is just fine.

Reply:
this should be done now.